fmlloader/src/main/java/net/minecraftforge/fml/loading/targets/CommonServerLaunchHandler.java

```diff
@@ -39,13 +39,21 @@ public LocatedPaths getMinecraftPaths() {
                       !path.startsWith("META-INF/"));
             }, mcextra
         );
+
+        // Todo in 1.20.2: Change the default filter to null instead of always true.
```

Comment:
We really should consider all the LaunchHandlers as internal API. But yes I can see this backwards compatibility for backports.

Reply:
Now that 1.20.2 is out, should I remove the back-compat in this PR but keep it in the backports? Or keep it as-is here too?

Reply:
Done